src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureBase {
```

Comment:
Don't begin class or method implementation with an empty line. It's a common mistake that can make the code look less clean. Please remove the empty line at the beginning of the class definition.